packages/shared/utils/is.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable antfu/top-level-function */
-export const isClient = typeof window !== 'undefined'
+export const isClient = typeof window !== 'undefined' && !!window.document
```

Comment:
updated it, or did you mean this?:

```suggestion
export const isClient = typeof window !== 'undefined' && typeof document !== 'undefined'
```

Reply:
Yes I think this would be better